code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -277,6 +277,12 @@
 			if(HAS_TRAIT(user, TRAIT_SPIRITUAL) && mind?.isholy)
 				msg += "[t_He] [t_has] a holy aura about [t_him].\n"
 				SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "religious_comfort", /datum/mood_event/religiously_comforted)
+			if(HAS_TRAIT(user, TRAIT_FAN_CLOWN) && mind?.isclowny)
+				msg += "[t_He] [t_has] a comical aura about [t_him].\n"
+				SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "comical_comfort", /datum/mood_event/comically_comforted)
+			if(HAS_TRAIT(user, TRAIT_FAN_MIME) && mind?.ismimey)
+				msg += "[t_He] [t_has] a silent aura about [t_him].\n"
+				SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "silent_comfort", /datum/mood_event/silently_comforted)

```

Comment:
clowns would be able to examine themselves thats silly. These quirks shouldnt be able to be taken by clowns and mimes imo

Reply:
Chaplains can currently examine themselves so I guess I should iron that out as well. As for disallowing mimes and clowns to have access to this, would something like how pacifism is cancelled with antag status work?

Reply:
Not sure, Adding a quirk blacklist on jobs would also not be a bad idea but im not sure what the best way of doing that is.

Reply:
Because of the way that quirk and job selection function it seems like the quirk can only be made unavailable post hoc.